qiskit/assembler/assemble_schedules.py

```diff
@@ -101,7 +101,13 @@ def _assemble_experiments(
     instruction_converter = instruction_converter(qobj.PulseQobjInstruction, **run_config.to_dict())
 
     formatted_schedules = [transforms.target_qobj_transform(sched) for sched in schedules]
-    compressed_schedules = transforms.compress_pulses(formatted_schedules)
+
+    frames_config = getattr(run_config, "frames_config", None)
+    resolved_schedules = [
```

Comment:
Can this be involved in `target_qobj_transform`?

Reply:
That works. See here: 582fdaf